app/helpers/me/notices_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Me::NoticesHelper
+  def dissmiss_all_notices_button
+    link_to_remote_with_confirm(:dismiss_all_notices.t,
```

Comment:
well, azul and I differ here. I like to use link_to_remote. I find it much more sensible than the new form. When we switch to jquery, we can just write our own link_to_remote function that calls link_to(..., :remote => true)